dir.traversal.targets

```diff
@@ -138,43 +138,41 @@
 
   <!-- FilterProjectsToTest will filter the project list according to the FilterToTestTFM value -->
   <Target Name="FilterProjectsToTest"
-          BeforeTargets="TestAllProjects" 
+          BeforeTargets="TestAllProjects"
           Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
 
+    <PropertyGroup>
+      <TestTFMSupportSet>$(FilterToTestTFM)</TestTFMSupportSet>
+      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='netcoreapp1.1'">$(TestTFMSupportSet)|netcoreapp1.0</TestTFMSupportSet>
+      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net461'">$(TestTFMSupportSet)|net46</TestTFMSupportSet>
+      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net462'">$(TestTFMSupportSet)|net461|net46</TestTFMSupportSet>
+      <TestTFMSupportSet Condition="'$(FilterToTestTFM)'=='net463'">$(TestTFMSupportSet)|net462|net461|net46</TestTFMSupportSet>
+    </PropertyGroup>
+
     <ItemGroup>
       <Project>
+        <!-- default to netcoreapp1.0.0 if TestTFMs aren't set on the project -->
+        <TestTFMs Condition="'%(Project.TestTFMs)'==''">netcoreapp1.0</TestTFMs>
```

Comment:
I'm not sure what an Any would look like in this case because we need to have a platform to run on, and that platform needs to be listed in the support clause for the tests. If we ever update this we will need to be sure to update all the test projects to add support clauses for whatever the new default is.